packages/components/tests/dummy/app/templates/components/badge-count.hbs

```diff
@@ -0,0 +1,118 @@
+  <p class="dummy-paragraph"><code class="dummy-code">BadgeCount</code>
+    is a numeric label used to display things like version number and collection enumeration in tabs.</p>
+  <p class="dummy-paragraph"><em>Notice: don’t use
+      <code class="dummy-code">BadgeCount</code>
+      for anything non-numeric, except version numbers (ie. “v1.2.0”).</em></p>
+</section>
+
+<section>
+  <h3 class="dummy-h3" id="bc-component-api"><a href="#bc-component-api" class="dummy-link-section">§</a>
+    Component API</h3>
+  <p class="dummy-paragraph" id="component-api-badge-count">Here is the API for the component:</p>
+  <dl class="dummy-component-props" aria-labelledby="component-api-badge-count">
+    <dt>size <code>enum</code></dt>
+    <dd>
+      <p>Acceptable values:</p>
+      <ol>
+        <li>small</li>
+        <li class="default">medium</li>
+        <li>large</li>
+      </ol>
+    </dd>
+    <dt>type <code>enum</code></dt>
+    <dd>
+      <p>Acceptable values:</p>
+      <ol>
+        <li class="default">filled</li>
+        <li>inverted</li>
+        <li>outlined</li>
+      </ol>
+    </dd>
+    <dt>color <code>enum</code></dt>
+    <dd>
+      <p>Acceptable values:</p>
+      <ol>
+        <li class="default">neutral</li>
+        <li>neutral-dark-mode</li>
+      </ol>
+    </dd>
+    <dt>text <code>string</code></dt>
+    <dd>
+      <p>The text value that should go in the badge counter.</p>
+    </dd>
+    <dt>...attributes</dt>
+    <dd>
+      <p><code class="dummy-code">...attributes</code> spreading is supported on this component.</p>
+    </dd>
+  </dl>
+</section>
+
+<section>
+  <h3 class="dummy-h3" id="bc-how-to-use"><a href="#bc-how-to-use" class="dummy-link-section">§</a> How to use</h3>
+  <p class="dummy-paragraph">Invocation of the component would look something like this:</p>
+  {{! prettier-ignore-start }}
+  <CodeBlock
+    @language="markup"
+    @code='
+      <Hds::BadgeCount @text="3" @size="large" @type="inverted" />
+    '
+  />
+  {{! prettier-ignore-end }}
+</section>
+<section>
+  <h3 class="dummy-h3" id="accessibility"><a href="#accessibility" class="dummy-link-section">§</a> Accessibility</h3>
+  <p class="dummy-paragraph">This component has been designed and implemented with accessibility in mind. When used as
+    recommended, there should not be any accessibility issues with this component.
+  </p>
+</section>
+
+<section data-test-percy>
+  <h3 class="dummy-h3" id="bc-showcase"><a href="#bc-showcase" class="dummy-link-section">§</a> Showcase</h3>
+
+  <h4 class="dummy-h4">Content</h4>
+  <div class="dummy-badge-base-sample">
+    <Hds::BadgeCount @text="3" />
+    <Hds::BadgeCount @text="99+" />
+    <Hds::BadgeCount @text="v1.2.3" />
+  </div>
+  <div class="dummy-badge-base-sample">
+    <p>This is a paragraph: <Hds::BadgeCount @text="3" /></p>
+  </div>
+
+  <h4 class="dummy-h4">Size</h4>
+  <div class="dummy-badge-base-sample">
+    {{#each @model.BADGE_COUNT_SIZES as |size|}}
```

Comment:
Now that you have moved the page, you also have to create a model for it and move the data in that file. Otherwise the page will not work (see the preview deploy: https://hds-components-git-jorytindall-badge-documenta-66637c-hashicorp.vercel.app/components/badge-count)

Search for `BADGE_COUNT_SIZES` in the entire codebase, and you will see that there is a file `packages/components/tests/dummy/app/routes/components/badge.js` that contains this data.

Essentially you have to:
- duplicate this file as `badge-count.js`
- remove from `badge-count.js` the `BADGE_SIZES/TYPES/COLORS` data
- remove from `badge.js` the `BADGE_COUNT_SIZES/TYPES/COLORS` data (not needed anymore)

If you have any problem, we can quickly pair on this.